bin/alpen-bridge/src/mode/operator.rs

```diff
@@ -49,6 +49,9 @@ pub(crate) async fn bootstrap(params: Params, config: Config) -> anyhow::Result<
         .secret_key()
         .await
         .map_err(|e| anyhow!("error while asking for p2p key: {e:?}"))?;
+
+    info!("Retrieved P2P secret key from S2: {sk:?}");
```

Comment:
This is still logging a secret key.
I don't think we should do this.
Maybe you can log the PK associated with this secret key?